backend/src/api/model/realm/mod.rs

```diff
@@ -198,60 +198,40 @@ impl Realm {
     /// (excluding both, the root realm and this realm). It starts with a
     /// direct child of the root and ends with the parent of `self`.
     async fn ancestors(&self, context: &Context) -> ApiResult<Vec<Realm>> {
-        let result = context.db
-            .query_raw(
-                "select id, parent, name, path_segment, full_path, index, child_order \
-                    from ancestors_of_realm($1) \
-                    where height <> 0 and id <> 0",
+        context.db
+            .query_mapped(
+                &format!(
+                    "select {} \
+                        from ancestors_of_realm($1) as ancestors \
+                        where height <> 0 and id <> 0",
+                    Self::col_names("ancestors"),
+                ),
```

Comment:
As you know, I like storing this in a variable to reduce the expression complexity. And then the whole `query_mapped` statement could fit into one line.

Reply:
Since it's not consistent already/anymore anyway, and I tend to slightly disagree, maybe we leave it for now? :thinking: :shrug: